### PR TITLE
Update travis script to build and deploy via heroku

### LINKED ISSUE #142: Configure continuous deployment for deploy branch
**Is your feature request related to a problem? Please describe.**
We need to configure continuous deployment for deploy branch. 

**Describe the solution you'd like**
Update the travis script and other necessary files

**Describe alternatives you've considered**
N/A

**Additional context**
N/A

### PR DESCRIPTION
**Pull request by Piumal1999**
## Purpose

The purpose of this PR is to fix #142 

## Goals

Triggering a build in backend when a pr is merged in frontend deploy branch

## Approach

Updated the travis script
Added the necessary config variables from travis dashboard
Used a script to trigger backend travis build by calling travis API (Added to the frontend).

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] I have read and understood the development best practices guidelines ( http://bit.ly/sef-best-practices )
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

## Related PRs
https://github.com/sef-global/scholarx-frontend/pull/129

## Test environment
 
Ubuntu 19.04

## Learning

N/A

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,20 +1,22 @@
...
+  - git clone https://github.com/sef-global/scholarx-frontend
+  - cd scholarx-frontend
+  - git checkout deploy
+  - npm install
+  - npm run build
+  - cd ../
 before_script:
-  - bash scripts/setup.sh
+  - cp scholarx-frontend/dist/bundle.js src/main/resources/static/dist/
+  - sudo rm -R scholarx-frontend
```
Comment (Gravewalker666) [inline .travis.yml:11]: Is it alright to add only the bundle.js file??
Don't we need to include the image files and an index.html file for this to work??

**Review by Gravewalker666 — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 1 above)
Comment (Piumal1999) [reply .travis.yml:11]: Yeah, you are correct. I'll add the other files

Comment (Piumal1999) [reply .travis.yml]: Updated

**Review by Gravewalker666 — APPROVED** [review]
Thanks @Piumal1999 Great work as always
Sorry for the late review

**Pull request merged**